packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -174,6 +174,15 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
    * @internal
    */
   protected _renderTask(): any {
+    const isEncrypted = this.props.resultConfiguration?.encryptionConfiguration !== undefined;
+
+    const EncryptionConfiguration = isEncrypted
+      ? {
+        EncryptionOption: this.props.resultConfiguration?.encryptionConfiguration?.encryptionOption,
+        KmsKey: this.props.resultConfiguration?.encryptionConfiguration?.encryptionKey,
```

Comment:
are we able to drop the `?` since we've already performed checks to verify that it's defined